server/devdocs/bin/generate-proptypes-index.js

```diff
@@ -4,31 +4,97 @@
  * This file generates an index of proptypes by component displayname, slug and folder name
  */
 
+const startTime = process.hrtime();
+
 /**
  * External Dependencies
  */
-
+require( 'babel-register' );
 const fs = require( 'fs' );
 const path = require( 'path' );
 const reactDocgen = require( 'react-docgen' );
+const { getPropertyName, getMemberValuePath, resolveToValue } = require( 'react-docgen/dist/utils' );
+const util = require( 'client/devdocs/docs-example/util' );
 
 const root = path.dirname( path.join( __dirname, '..', '..' ) );
 const pathSwap = new RegExp(path.sep, 'g');
+const handlers = [ ...reactDocgen.defaultHandlers, commentHandler ];
```

Comment:
I moved handlers out of the loop and to the top of the file ... saved quite a bit of time :)